strings/revenant_names.json

```diff
@@ -0,0 +1,51 @@
+{
+    "spirit_type": [
+        "Spirit",
+        "Ghost",
+        "Spectre",
+        "Phantom",
+        "Revenant",
+        "Essence",
+        "Soul"
+    ],
+
+    "adverb": [
```

Comment:
These are actually adjectives 😮

Reply:
Can we add "hilarious" to this list? You have "laughter" as a possible theme. inb4 clown ghost

Reply:
Adverbs turned into adjectives, thanks. Hilarious added as a possible adjective, we have one "lighter" theme, so I guess it alright if we have also one lighter adjective.